iati/core/schemas.py

```diff
@@ -24,15 +24,15 @@ class Schema(object):
 
     """
 
-    def __init__(self, name=None):
+    root_element_name = ''
```

Comment:
In terms of being a constant or not... I s'pose this is more applicable to the child classes which have fixed root element names that shouldn't change.